charts/cluster-scanner/values.yaml

```diff
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
```

Comment:
This should be grouped in a single component as `rsi` and `ise`?

Can we -at least- try to have a nested structure for `eve` like:
```
eve:
    enabled: false
    integration:
        enabled: false
```

Reply:
I was following Helm Chart [suggested best practices](https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values) about values and avoiding unnecessary nesting

if you feel like it needs nesting, I can surely change it

---

regarding grouping/putting config inside `rsi` and `ise`, I was trying to put in there the bare minimum set of parameters that needs to be scoped to a single container:
end-users won't care about which container does what, but just enable/disable EVE, or the cache, etc.

So, unless strictly required (eg: image name/tag) I would avoid requiring end-user to know how we split responsibilities into the applications/containers.

Does it make sense to you? @mavimo